core/cloudflow-cli/src/main/scala/akka/cli/cloudflow/execution/DeployExecution.scala

```diff
@@ -136,7 +135,7 @@ final case class DeployExecution(d: Deploy, client: KubeClient, logger: CliLogge
       Failure(CliException("The application specification doesn't contains deployments"))
     } else {
       // Get the first available image, all images must be present in the same repository.
-      val imageRef = crApp.spec.deployments(0).image
+      val imageRef = crApp.spec.deployments.head.image
```

Comment:
should we do `headOption` instead and check for failures if `None` to wrap in some `CliException` ?